Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Use Python 3.12 slim image
+FROM python:3.12-slim-bookworm
+
+# Set environment variables
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    PIP_NO_CACHE_DIR=1 \
+    PIP_DISABLE_PIP_VERSION_CHECK=1
+
+# Create app user and group
+RUN groupadd --gid 1000 app && \
+    useradd --uid 1000 --gid app --shell /bin/bash --create-home app
+
+# Install system dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        curl \
+        git \
+        build-essential \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set working directory
+WORKDIR /app
+
+# Copy requirements and install Python dependencies
+COPY requirements-dev.txt .
+RUN pip install --no-cache-dir -r requirements-dev.txt
+
+# Copy source code
+COPY src/ ./src/
+COPY pyproject.toml .
+COPY README.md .
+COPY LICENSE .
+
+# Install the package in development mode
+RUN pip install -e .
+
+# Change ownership to app user
+RUN chown -R app:app /app
+
+# Switch to app user
+USER app
+
+# Expose ports
+EXPOSE 2218
+
+# Health check
+HEALTHCHECK --interval=30s --timeout=5s --start-period=30s --retries=3 \
```

Comment:
I believe if the healthcheck is defined in the dockerfile, it does not need to be in the docker compsoe file or devservices config.yml file

Reply:
is there a preference of which location? If not, I'll defer to just keep the one in `devservices/config.yml`

Reply:
synced offline, removed it from Dockerfile